.travis.yml

```diff
@@ -27,7 +30,9 @@ env:
 #- make dep-install
 #- make dep-update
 
-script: make test
+script:
+  - test -z $(gofmt -s -l $GO_FILES) # Check if fail hasn't been formatted with gofmt
```

Comment:
can you explain? The comment "Check if fail hasn't been formatted with gofmt" is confusing me.

Though I think this comment is just a typo. Do you mean "Fail if files have not been formatted with gofmt" ?

Reply:
Yes sorry, It was an error. The point is If we should add a task in Travis for gofmt.

Reply:
Yes, I think we can add this to .travis. I think its fine to fail a build if the dev didn't even do a simple gofmt (and once we have a git hook, even that won't be required for the dev to do).